shared/actions/teams.js

```diff
@@ -809,10 +820,17 @@ function* getTeams(state) {
     if (err.code === RPCTypes.constantsStatusCode.scapinetworkerror) {
       // Ignore API errors due to offline
     } else {
+      yield Saga.put(
+        WaitingGen.createDecrementWaiting({
+          error: err,
+          key: Constants.teamsLoadedWaitingKey,
+        })
+      )
+
       throw err
     }
   } finally {
-    yield Saga.put(TeamsGen.createSetLoaded({loaded: true}))
+    yield Saga.put(WaitingGen.createDecrementWaiting({key: Constants.teamsLoadedWaitingKey}))
```

Comment:
if this and the `decrementWaiting` in the else block above both happen, do we end up in a bad state?

Reply:
we shouldn't be doing this explicitly, the rpc will already decrement right?

Reply:
we basically never need manual dispatches of the waiting keys

Reply:
This was to copy the previous behavior of having a loaded value, but now that you mention it. I think we can remove that behavior completely and just rely on the engine